src/index.js

```diff
@@ -1,14 +1,39 @@
-import _ from 'lodash';
 import './style.css';
 
- function component() {
-   const element = document.createElement('div');
+const arr = [
+  {
+    description: 'Study JS',
+    completed: '',
+    index: 1,
+  },
+  {
+    description: 'Study HTML',
+    completed: '',
+    index: 2,
+  },
+  {
+    description: 'Study CSS',
+    completed: 'checked',
+    index: 3,
+  },
+];
```

Comment:
- [ ] Kindly consider changing the value of `completed` key from a `string` to a boolean `false` preferably because you would implement `clear all` functionality based on this boolean later.
  - Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should 
  contain three keys:
  description [string].
  completed [bool].
  index: [number].